pkgs/tools/misc/diskonaut/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, rustPlatform, fetchFromGitHub }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "diskonaut";
+  version = "1.15.1";
+
+  src = fetchFromGitHub {
+    owner = "imsnif";
+    repo = "diskonaut";
+    rev = "f716f629c0a2c5e9c20e5e81c6ba260b357c9ed3";
+    sha256 = "0r2f3rljq82c8vzmg9anwxiz3bbsn1n4g9vmnda4hg9kxg96v26a";
```

Comment:
```suggestion
    rev = version;
    sha256 = "0vnmch2cac0j9b44vlcpqnayqhfdfdwvfa01bn7lwcyrcln5cd0z";
```

Reply:
Is there any significant difference/reason I should use version over `${version}`?
Am I just hallucinating by remembering seeing `${version}` used before?

Reply:
First of all, since `version` is already a string, they are equivalent, so `version == "${version}"`. However, the former is simpler and the latter is more work for the interpreter (since it has to process the antiquotation).

In the case of versions using a string with antiquotation can be useful when the tag contains the version number as an affix. For example, many projects prefix git version tags with *v*, so in that case you could use:

```
rev = "v${version}";
```

Reply:
> First of all, since `version` is already a string, they are equivalent, so `version == "${version}"`. However, the former is simpler and the latter is more work for the interpreter (since it has to process the antiquotation).
> 
> 
> 
> In the case of versions using a string with antiquotation can be useful when the tag contains the version number as an affix. For example, many projects prefix git version tags with *v*, so in that case you could use:
> 
> 
> 
> ```
> 
> rev = "v${version}";
> 
> ```

Ahhh, `v${version}` is the exact case I was thinking of.
The dots in my head just sort of fail to connect at times 🤪
Thanks!